src/clients/api/mutations/useRedeemUnderlying.ts

```diff
@@ -8,23 +8,24 @@ import {
   RedeemUnderlyingOutput,
 } from 'clients/api';
 import FunctionKey from 'constants/functionKey';
-import { useTokenContract } from 'clients/contracts/hooks';
+import { useVTokenContract } from 'clients/contracts/hooks';
 
 const useRedeemUnderlying = (
-  { assetId }: { assetId: VTokenId },
+  { assetId, account }: { assetId: VTokenId; account: string | undefined },
```

Comment:
I think we should always have `account` as required and defined for mutations that require it. I checked the `redeem` function and although it allows `account` to be a `string` or `undefined`, it then asserts it is defined using an exclamation mark. That's risky, because it silents TypeScript and so can lead to runtime errors.

For cases where `account` can be undefined but we still need to render the hook, we should use the `enabled` prop:
```tsx
useRedeemUnderlying({
  assetId,
  account: accountAddress || ''
}, {
  enabled: !!account
});
```